presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -286,6 +286,8 @@ Name                                        Description
 ``running-mode``                            Set to ``query-bank`` to make the Verifier run in ``query-bank`` mode. Supports
                                             ``query-bank`` and ``control-test``. Defaults to ``control-test``.
 ``save-snapshot``                           Set to ``true`` to save checksums to ``mysql`` database.
+``extended-verification``                   Set to ``true`` to run extended verification. It only applies to ``Insert`` and ``CreateTableAsSelect`` queries.
```

Comment:
Would it be beet to have "writer" word in it? Or any other that would hint on the nature of the extended verification?